txjuju/tests/test___init__.py

```diff
@@ -22,3 +25,30 @@ def test_unsupported(self):
         """Passing in an unrecognized release should cause an error."""
         with self.assertRaises(ValueError):
             txjuju.get_cli_class("<???>")
+
+
+class PrepareForBootstrapTests(unittest.TestCase):
+
+    def setUp(self):
+        super(PrepareForBootstrapTests, self).setUp()
+        self.cfgdir = tempfile.mkdtemp(prefix="txjuju-test-")
+
+    def tearDown(self):
+        shutil.rmtree(self.cfgdir)
+        super(PrepareForBootstrapTests, self).tearDown()
+
+    def test_juju2(self):
```

Comment:
I know the tests are small and easy to understand, but can we add a docstring comment explaining the why of the intent of this test. Generally speaking it'd be nice to have a oneliner dosctring on most unit tests as the unit test name may leave something to be desired as far as context. The rest of the test code in txjuju seems to adhere to a docstring per unit test.